packages/date-picker/src/DatePicker/components/Calendar/Calendar.scss

```diff
@@ -12,14 +12,12 @@ $cell-border-radius: 3px;
   background-color: $color-white;
   z-index: 1010;
 
-  .wrapper {
-    padding-bottom: 0;
+  .container {
```

Comment:
Perhaps we should refactor this file later. The nesting isn't necessary since CSS modules ensures the classes are unique and don't clash with global ones.